lib/Scheduler/aco.cpp

```diff
@@ -536,9 +567,9 @@ void ACOScheduler::writePheromoneGraph(std::string Stage) {
       OutPath + "/" + dataDepGraph_->GetDagID() + "@" + Stage + ".dot";
   FILE *Out = fopen(FullOutPath.c_str(), "w");
   if (!Out) {
-    Logger::Info("Could now open file to write pheromone display at %s."
-                 " Skipping.",
-                 FullOutPath.c_str());
+    Logger::Error("Could now open file to write pheromone display at %s."
+                  " Skipping.",
+                  FullOutPath.c_str());
```

Comment:
I think we wanted to use `llvm::report_fatal_error` for this. It would be something like `llvm::report_fatal_error(llvm::Twine("Could ... Skipping. File: ").concat(FullOutPath), false)` or `llvm::report_fatal_error("Could ... Skipping. File: " + FullOutPath, false)`.

Also, there was a typo: "Could **now**" --> "Could _not_"

Reply:
I used Logger::Error, because this is not a fatal error.  This should not crash the program.  I just wanted to inform the user that an error occurred.  Since this function only provides debugging information regarding the pheromone table the scheduler is able to continue functioning, albeit without extra debugging info.

Reply:
Thanks, I goofed when reading the code.